src/templates/dataGridTable.jsx

```diff
@@ -3,10 +3,43 @@ import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import DataGridTableRow from './dataGridTableRow.js';
-import DragListener from '../utils/dragListener.js';
+import DataGridTableRow from './dataGridTableRow';
```

Comment:
We are deliberately going with no file extensions now?  When did that change?

Reply:
Yes, but only for react-cm-ui. The decision was made because of switching to `.jsx` extensions and Babel isn't taking a `.jsx` file and spitting out babelfied file with the same extension. In other words, it's an easy fix.

Reply:
Understood (well, understood-ish) ... it solved a problem. 👍